lib/escape-url.js

```diff
@@ -0,0 +1,13 @@
+module.exports = function(url) {
```

Comment:
`lib/escape-url-js` => `lib/url/escape.js` 

`module.exports = function escape (url) {`

So we hopefully can require the named `{Function}` without having to decl a new variable later :)

Reply:
Done.